.semgrep/tests/sol-safety-use-disable-initializers.t.sol

```diff
@@ -0,0 +1,61 @@
+// Semgrep tests for Solidity rules are defined in this file.
```

Comment:
You will have to rename this file to sol-rules.sol-safety-use-disable-initializers.t.sol for it to work

Reply:
why is that?

Reply:
i'm guessing it's because that's the name of the file the rules are defined in `sol-rules`